### PR TITLE
[7.4 only] LPS-123680 buildLang

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.json [hunk 1]
```diff
@@ -1,3 +1,82 @@
...
+							{
+								"label": "full-width",
+								"value": "dropdown-full"
+							}
+						]
+					}
+				},
+				{
+					"defaultValue": false,
+					"label": "Keep panel open in edit mode",
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.json:75]: This should be a language key isn't it?

**Review by victorg1991 — COMMENTED** [review]
Looking super good so far 😍

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.json (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.json]: Yep! Done 🙂 thanks bro

**Comment by sandrodw3** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
sent here: https://github.com/liferay-echo/liferay-portal/pull/3054

**Pull request closed without merging**